osquery/tables/system/posix/augeas.cpp

```diff
@@ -10,6 +10,9 @@
 
 #include <augeas.h>
 
+#include <sstream>
```

Comment:
nit, if you nix the string stream for the string and `operator+` I mentioned, then nix this too.